e2e/tests/1.21/upgrade/07-write-data.yaml

```diff
@@ -0,0 +1,5 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - command: ../../../../hack/write-pod-data.sh pod-1 /mnt/storageosupgrade StorageOS
```

Comment:
Is there any way to know if this failed?

As we don't have an assert for it

Reply:
Ah I see now that we later read the data and check it matches

Reply:
If a kuttl command fails to execute, the test-step will automatically fail and exit from what I've seen.